modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/info/item/provider/LayoutInfoItemFieldValuesProvider.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.layout.admin.web.internal.info.item.provider;
+
+import com.liferay.info.field.InfoFieldValue;
+import com.liferay.info.item.InfoItemFieldValues;
+import com.liferay.info.item.provider.InfoItemFieldValuesProvider;
+import com.liferay.info.localized.InfoLocalizedValue;
+import com.liferay.layout.admin.web.internal.info.item.LayoutInfoItemFields;
+import com.liferay.portal.kernel.model.Layout;
+import com.liferay.portal.kernel.util.LocaleUtil;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Adolfo Pérez
+ */
+@Component(service = InfoItemFieldValuesProvider.class)
+public class LayoutInfoItemFieldValuesProvider
+	implements InfoItemFieldValuesProvider<Layout> {
+
+	@Override
+	public InfoItemFieldValues getInfoItemFieldValues(Layout layout) {
+		return InfoItemFieldValues.builder(
+		).infoFieldValue(
+			new InfoFieldValue<>(
+				LayoutInfoItemFields.nameInfoField,
```

Comment:
same comment about field order by name

Reply:
The order here affects the order in the UI, this is intentional.